solutions/observability/apps/managed-intake-service-event-api.md

```diff
@@ -4476,40 +4475,4 @@ The managed intake service uses JSON Schema to validate requests. The specificat
 }
 ```
 
-::::
-
-
-
-## OpenTelemetry API [observability-apm-server-api-opentelemetry-api] 
```

Comment:
I'm assuming this isn't relevant with EDOT, but I could be wrong. I'd appreciate any input here. I didn't see it documented in the new GH docs.